amqp/src/main/java/io/github/tcdl/msb/adapters/amqp/AmqpAdapterFactory.java

```diff
@@ -96,6 +94,19 @@ protected ConnectionFactory createConnectionFactory(AmqpBrokerConfig adapterConf
         if(virtualHost.isPresent()) {
             connectionFactory.setVirtualHost(virtualHost.get());
         }
+
+        try {
+            if (adapterConfig.useSSL()) {
+                LOG.info("Configuring adapter to use SSL...");
```

Comment:
I would still use info. These log statements trace the process of application initialization. There's no harm in having 2 messages performance wise bit it may simplify investigation of a problem